components/List/index.js

```diff
@@ -0,0 +1,20 @@
+import React, {Component} from 'react'
+import {StyleSheet, View} from 'react-native'
+
+export default class List extends Component {
+    render() {
+        return (
+            <View style = styles.list>
+                <h3>{this.props.title}</h3>
+                {this.props.cards.map((card, i) => <Card key={i} {...card} />)}
```

Comment:
this is technically incorrect, since you can't rely on card order for keys here. cards have `id`s, so you should use those for the keys